tests/Agent/IntegrationTests/ContainerApplications/AwsSdkTestApp/AwsSdkExercisers/AwsSdkDynamoDBExerciser.cs

```diff
@@ -0,0 +1,249 @@
+// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using Amazon.DynamoDBv2.Model;
+using Amazon.DynamoDBv2;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using System;
+using System.Collections.Generic;
+using Amazon.Runtime;
+using System.Threading;
+
+namespace AwsSdkTestApp.AwsSdkExercisers
+{
+    public class AwsSdkDynamoDBExerciser : IDisposable
+    {
+        private readonly AmazonDynamoDBClient _amazonDynamoDBClient;
+
+        public AwsSdkDynamoDBExerciser()
+        {
+            _amazonDynamoDBClient = GetDynamoDBClient();
+        }
+        
+
+        private AmazonDynamoDBClient GetDynamoDBClient()
+        {
+
+            AmazonDynamoDBConfig clientConfig = new AmazonDynamoDBConfig();
+            // Set the endpoint URL
+            clientConfig.ServiceURL = "http://dynamodb:8123"; // port must match what is set in docker compose
+            clientConfig.AuthenticationRegion = "us-west-2";
+            var creds = new BasicAWSCredentials("xxx", "xxx");
```

Comment:
Is this needed? IIRC there's a constructor that doesn't require a credentials object

Reply:
When I was getting the plumbing set up earlier this week, I could not get the DynamoDB calls to succeed without providing dummy credentials.  Searching online for the error I got (something about not being able to find credentials) indicated that when working with a local test DynamoDB that you had to specify some creds but it didn't matter what they were.  The same thing is being done in the SQS exerciser, for what it's worth.